feature-tools give an error

$ feature-tools -u http://feature.nmdp-bioinformatics.org/ -i requests.txt -o features.txt

results in:
retrofit.RetrofitError: com.fasterxml.jackson.databind.JsonMappingException: No suitable constructor found for type [simple type, class org.nmdp.service.feature.Feature]: can not instantiate from JSON object (need to add/enable type information?)
 at [Source: retrofit.ExceptionCatchingTypedInput$ExceptionCatchingInputStream@5e9ec769; line: 2, column: 3]
	at retrofit.RetrofitError.conversionError(RetrofitError.java:33)
	at retrofit.RestAdapter$RestHandler.invokeRequest(RestAdapter.java:383)
	at retrofit.RestAdapter$RestHandler.invoke(RestAdapter.java:240)
	at com.sun.proxy.$Proxy11.createFeature(Unknown Source)
	at org.nmdp.service.feature.tools.FeatureTools.call(FeatureTools.java:111)
	at org.nmdp.service.feature.tools.FeatureTools.main(FeatureTools.java:178)
Caused by: retrofit.converter.ConversionException: com.fasterxml.jackson.databind.JsonMappingException: No suitable constructor found for type [simple type, class org.nmdp.service.feature.Feature]: can not instantiate from JSON object (need to add/enable type information?)
 at [Source: retrofit.ExceptionCatchingTypedInput$ExceptionCatchingInputStream@5e9ec769; line: 2, column: 3]
	at retrofit.converter.JacksonConverter.fromBody(JacksonConverter.java:40)
	at retrofit.RestAdapter$RestHandler.invokeRequest(RestAdapter.java:367)
	... 4 more
Caused by: com.fasterxml.jackson.databind.JsonMappingException: No suitable constructor found for type [simple type, class org.nmdp.service.feature.Feature]: can not instantiate from JSON object (need to add/enable type information?)
 at [Source: retrofit.ExceptionCatchingTypedInput$ExceptionCatchingInputStream@5e9ec769; line: 2, column: 3]
	at com.fasterxml.jackson.databind.JsonMappingException.from(JsonMappingException.java:148)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1063)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:264)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:124)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3051)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2206)
	at retrofit.converter.JacksonConverter.fromBody(JacksonConverter.java:36)
	... 5 more

--- a/resource/src/main/java/org/nmdp/service/feature/resource/FeatureMixIn.java
+++ b/resource/src/main/java/org/nmdp/service/feature/resource/FeatureMixIn.java
@@ -37,6 +37,7 @@ public final class FeatureMixIn {
     public FeatureMixIn(final @JsonProperty("locus") String locus,
                         final @JsonProperty("term") String term,
                         final @JsonProperty("rank") int rank,
+                        final @JsonProperty("accession") long accession,                        
                         final @JsonProperty("sequence") String sequence) {
 
         // empty

--- a/resource/src/test/java/org/nmdp/service/feature/resource/FeatureMixInTest.java
+++ b/resource/src/test/java/org/nmdp/service/feature/resource/FeatureMixInTest.java
@@ -33,6 +33,6 @@ public final class FeatureMixInTest {
 
     @Test
     public void testConstructor() {
-        assertNotNull(new FeatureMixIn("locus", "term", 2, "ACTG"));
+        assertNotNull(new FeatureMixIn("locus", "term", 2, 1, "ACTG"));
     }
 }

